src/VisualStudio/InteractiveServices/Interactive/VsInteractiveWindowProvider.cs

```diff
@@ -58,7 +58,7 @@ internal abstract class VsInteractiveWindowProvider
         protected abstract Guid LanguageServiceGuid { get; }
         protected abstract Guid Id { get; }
         protected abstract string Title { get; }
-        protected abstract void LogSession(string key, string value);
+        protected abstract void LogSession(string key, string value, params object[] objects);
```

Comment:
I think the difference between derived type is the `FunctionId` passed to `Logger.Log`.